modules/telemetry-showing-data-collected-from-the-cluster.adoc

```diff
@@ -6,28 +6,152 @@
 [id="showing-data-collected-from-the-cluster_{context}"]
 = Showing data collected by Telemetry
 
-You can see the cluster and components time series data captured by Telemetry.
+You can view the cluster and components time series data captured by Telemetry.
 
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* You must log in to the cluster with a user that has either the `cluster-admin` role or the `cluster-monitoring-view` role.
+* You have installed the {product-title} CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role or the `cluster-monitoring-view` role.
 
 .Procedure
 
-. Find the URL for the Prometheus service that runs in the {product-title} cluster:
+* Run the following command, which queries a cluster's Prometheus service and returns the full set of time series data captured by Telemetry:
 +
 [source,terminal]
 ----
-$ oc get route prometheus-k8s -n openshift-monitoring -o jsonpath="{.spec.host}"
+$ curl -G -k -H "Authorization: Bearer $(oc whoami -t)" \
```

Comment:
```
odf_system_bucket_count
odf_system_objects_total
```
these are just example metrics between 4.11 and 4.12, actually we also have other metrics in 4.12, but not in 4.11
4.12: https://github.com/openshift/cluster-monitoring-operator/blob/release-4.12/manifests/0000_50_cluster-monitoring-operator_04-config.yaml
4.11: https://github.com/openshift/cluster-monitoring-operator/blob/release-4.11/manifests/0000_50_cluster-monitoring-operator_04-config.yaml

metrics list in 4.12 but not in 4.11
```

job:odf_system_pvs:count

namespace:noobaa_accounts:max
namespace:noobaa_buckets_claims:max
namespace:noobaa_namespace_buckets:max
namespace:noobaa_namespace_resources:max
namespace:noobaa_system_health_status:max
namespace:noobaa_unhealthy_bucket_claims:max
namespace:noobaa_unhealthy_namespace_buckets:max
namespace:noobaa_unhealthy_namespace_resources:max
namespace:noobaa_usage:max

ocs_advanced_feature_usage
{__name__="odf_system_bucket_count", system_type="OCS", system_vendor="Red Hat"}
odf_system_health_status
{__name__="odf_system_bucket_count", system_type="OCS", system_vendor="Red Hat"}
odf_system_raw_capacity_total_bytes
odf_system_raw_capacity_used_bytes

os_image_url_override:sum
platform:hypershift_hostedclusters:max
platform:hypershift_nodepools:max
```
not sure if we have a better solution for the doc